[bitnami/redis-cluster] Update redis-cluster-env.sh

byErikas opened the pull request:
Adding file reading support for missing enviroment variable `REDISCLI_AUTH`

### Description of the change
- Added `REDISCLI_AUTH` to `redis_cluster_env_vars` array inside `/opt/bitnami/scripts/redis-cluster-env.sh`
- Added an `export REDIS_CLI` command to the same script as above to keep it set

### Benefits
- Allows for the ability to use `REDISCLI_AUTH` with a file instead of a variable
- Provides consistency when used with `REDIS_PASSWORD_FILE`

### Possible drawbacks
- Worth noting I do not know if this is omitted on purpose or just got forgotten, if omitted on purpose this may have breaking changes

### Applicable issues
- fixes #35982

byErikas committed:
Update redis-cluster-env.sh

Adding file read support for enviroment variable of "REDISCLI_AUTH"

Signed-off-by: Erikas Byštautas <55389436+byErikas@users.noreply.github.com>

fevisera commented:
Hi @byErikas, 

Thank you for your contribution! 

Please, include in the PR the changes related to the other redis-cluster version, `6.2`.

Thanks.

byErikas committed:
Add changes to 6.2 version

Added REDISCLI_AUTH variable file support for version  redis-cluster version 6.2

Signed-off-by: Erikas Byštautas <55389436+byErikas@users.noreply.github.com>

byErikas commented:
I've added changes for the `6.2` version, please take a look at it when you can

fevisera reviewed (approved):
LGTM!

Thank you @byErikas for updating the PR and for your contribution!
